docs/api/doc-block-story.md

```diff
@@ -0,0 +1,164 @@
+```
+<!-- prettier-ignore-end -->
+
+##### `of`
+
+Type: Story export
+
+Specifies which story is rendered by the `Story` block. If no `of` is defined and the MDX file is [attached](./doc-block-meta.md#attached-vs-unattached), the primary (first) story will be rendered.
+
+##### `args` (deprecated)
+
+Type: `Partial<TArgs>`
+
+Defining and configuring stories in MDX is deprecated. See the [Migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files) for details.
+
+##### `argTypes` (deprecated)
+
+Type: `Partial<ArgTypes<TArgs>>`
+
+Defining and configuring stories in MDX is deprecated. See the [Migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files) for details.
+
+##### `decorators` (deprecated)
+
+Type: `DecoratorFunction<TRenderer, TArgs>[]`
+
+Defining and configuring stories in MDX is deprecated. See the [Migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files) for details.
+
+##### `loaders` (deprecated)
+
+Type: `LoaderFunction<TRenderer, TArgs>[]`
+
+Defining and configuring stories in MDX is deprecated. See the [Migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files) for details.
+
+##### `name` (deprecated)
+
+Type: `StoryName`
+
+Defining and configuring stories in MDX is deprecated. See the [Migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files) for details.
+
+##### `parameters` (deprecated)
+
+Type: `Parameters`
+
+Defining and configuring stories in MDX is deprecated. See the [Migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files) for details.
+
+##### `play` (deprecated)
+
+Type: `PlayFunction<TRenderer, TArgs>`
+
+Defining and configuring stories in MDX is deprecated. See the [Migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files) for details.
+
+##### `render` (deprecated)
+
+Type: `ArgsStoryFn<TRenderer, TArgs>`
+
+Defining and configuring stories in MDX is deprecated. See the [Migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files) for details.
+
+##### `story` (deprecated)
+
+Type: `Omit<StoryAnnotations<TRenderer, TArgs>, 'story'>`
+
+Defining and configuring stories in MDX is deprecated. See the [Migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files) for details.
+
+##### `storyName` (deprecated)
+
+Type: `StoryName`
+
+Defining and configuring stories in MDX is deprecated. See the [Migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files) for details.
+
+##### `tags` (deprecated)
+
+Type: `Tag[]`
+
+Defining and configuring stories in MDX is deprecated. See the [Migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files) for details.
```

Comment:
I don't think we need to document this "deprecated" prop, given that tags is only being introduced in 7.0.

```suggestion
```